### PR TITLE
Add CMA 2.0.2 upgrade notes to Core docs

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2751:](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2751)

## Changes

* Update core docs to add CMA 2.0.2 upgrade instructions

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: docs/upgrade-notes/update-cma-2.0.2.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        "version": "$LATEST",
+        "arn": "arn:aws:lambda:us-east-1:xxxx:function:jk-tf-QueueGranules"
+      }
+```
+
+### Actions Required
+
+The following should be done as part of a Cumulus stack update to utilize `cumulus message adapter` > `2.0.2`:
+
+- Python tasks that utilize [`cumulus-message-adapter-python`](https://github.com/nasa/cumulus-message-adapter-python) should be updated to use > `2.0.0`, their lambdas rebuilt and Cumulus workflows reconfigured to use the updated version.
```
Comment (flamingbear) [inline docs/upgrade-notes/update-cma-2.0.2.md:57]:
```suggestion
- Python tasks that utilize [`cumulus-message-adapter-python`](https://github.com/nasa/cumulus-message-adapter-python) should be updated to use > `2.0.2`, their lambdas rebuilt and Cumulus workflows reconfigured to use the updated version.
```

File: docs/upgrade-notes/update-cma-2.0.2.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- Python activities that utilize [`cumulus-process-py`](https://github.com/nasa/cumulus-process-py) should be rebuilt using > `1.0.0` with updated dependencies, and have their images deployed/Cumulus configured to use the new version.
```
Comment (flamingbear) [inline docs/upgrade-notes/update-cma-2.0.2.md:59]: Does this have a specific version?

File: docs/upgrade-notes/update-cma-2.0.2.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+id: update-cma-2.0.2
+title: Upgrade to CMA 2.0.2
+hide_title: false
+---
+
+## Updating a Cumulus Deployment to CMA 2.0.2
+
+### Background
+
+The Cumulus Message Adapter has been updated in [release 2.0.0](https://github.com/nasa/cumulus-message-adapter/releases/tag/v2.0.0) to no longer utilize the AWS step function API to look up the defined name of a step function task for population in meta.workflow_tasks, but instead use an incrementing integer field.
```
Comment (flamingbear) [inline docs/upgrade-notes/update-cma-2.0.2.md:11]:
```suggestion
The Cumulus Message Adapter has been updated in [release 2.0.2](https://github.com/nasa/cumulus-message-adapter/releases/tag/v2.0.0) to no longer utilize the AWS step function API to look up the defined name of a step function task for population in meta.workflow_tasks, but instead use an incrementing integer field.
```

**Review by flamingbear — CHANGES_REQUESTED** [review]
Two suggestions, to remove 2.0.0 in favor of 2.0.2.
one question about `cumulus-process-py`'s verision.

File: docs/upgrade-notes/update-cma-2.0.2.md (hunk 1 above)
Comment (Jkovarik) [reply docs/upgrade-notes/update-cma-2.0.2.md:57]: That's not right - this is cumulus-message-adapter-python, which has a version that is decoupled from the CMA itself.

File: docs/upgrade-notes/update-cma-2.0.2.md (hunk 2 above)
Comment (Jkovarik) [reply docs/upgrade-notes/update-cma-2.0.2.md:59]: We can specify v1.0.0, however by the time users actually get around to this that might not be true. 

This is one of the reasons I'm a bit uncomfortable putting this in upgrade docs, given it's not tied to a particular Cumulus version.

File: docs/upgrade-notes/update-cma-2.0.2.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+The following should be done as part of a Cumulus stack update to utilize `cumulus message adapter` > `2.0.2`:
+
+- Python tasks that utilize [`cumulus-message-adapter-python`](https://github.com/nasa/cumulus-message-adapter-python) should be updated to use > `2.0.0`, their lambdas rebuilt and Cumulus workflows reconfigured to use the updated version.
+
+- Python activities that utilize [`cumulus-process-py`](https://github.com/nasa/cumulus-process-py) should be rebuilt using > `1.0.0` with updated dependencies, and have their images deployed/Cumulus configured to use the new version.
+
+- The [`cumulus-message-adapter`](https://github.com/nasa/cumulus-message-adapter) v2.0.2 lambda layer should be made available in the deployment account, and the Cumulus deployment should be reconfigured to use it (via the `cumulus_message_adapter_lambda_layer_version_arn` variable in the `cumulus` module).  This should address all Core node.js tasks that utilize the CMA, and many contributed node.js/JAVA components.
+
+Once the above have been done, redeploy Cumulus to apply the configuration and the updates should be live.
```
Comment (markdboyd) [inline docs/upgrade-notes/update-cma-2.0.2.md:63]: This all looks good - but we should probably point people to these upgrade notes in the "migration steps" section of the CHANGELOG

File: docs/upgrade-notes/update-cma-2.0.2.md (hunk 2 above)
Comment (flamingbear) [reply docs/upgrade-notes/update-cma-2.0.2.md:59]: I was confusing it with another package I suppose.

File: docs/upgrade-notes/update-cma-2.0.2.md (hunk 4 above)
Comment (Jkovarik) [reply docs/upgrade-notes/update-cma-2.0.2.md:63]: I'm not sure I can agree - the updates here are anecdotal to Core *particularly* to a specific release except that we aren't fully testing all prior backports.    A CHANGELOG entry might be warranted for v10 to note that we documented the process, but there are no changes to the Core project itself.    

That said, it makes total sense we need to communicate this to the user community outside the post in the CMA release notes, as all of these components function together as a system.

Comment (markdboyd) [reply docs/upgrade-notes/update-cma-2.0.2.md:63]: Fair enough

**Review by flamingbear — APPROVED** [review]

**Pull request merged**